HTAN.model.csv

```diff
@@ -357,7 +357,7 @@ Slide Charge Type,A description of the charge on the glass slide.,"Uncharged, Ch
 Section Thickness Value,"Numeric value to describe the thickness of a slice to tissue taken from a biospecimen, measured in microns (um).",,,,TRUE,Biospecimen,,https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search?publicId=3024951&version=1.0,
 Sectioning Days from Index,Number of days from the research participant's index date that the biospecimen was sectioned after collection,,,,TRUE,Biospecimen,,,int
 Storage Method,The method by which a biomaterial was stored after preservation or before another protocol was used.,"Ambient temperature, Cut slide, Fresh, Frozen at -70C, Frozen at -80C, Frozen at -150C, Frozen in liquid nitrogen, Frozen in vapor phase, Paraffin block, RNAlater at 4C, RNAlater at 25C, RNAlater at -20C, Refrigerated at 4 degrees, Refrigerated vacuum chamber, 4C in vacuum chamber, Desiccant at 4C, Not Applicable, unknown",,,TRUE,Biospecimen,,https://data.humancellatlas.org/metadata/dictionary/biomaterial/specimen_from_organism#specimen_from_organism-preservation_storage-storage_method,
-Processing Days from Index,Number of days from the research participant's index date that the biospecimen was processed,,,,TRUE,Biospecimen,,,regex match \d+$|Not\sapplicable$|unknown$
+Processing Days from Index,Number of days from the research participant's index date that the biospecimen was processed,,,,TRUE,Biospecimen,,,regex match \d+$|Not\sapplicable$|unknown$|NA$

```

Comment:
Does adding NA make sense when Not applicable is valid. Going to revisit this in the parent issue